### ISSUE TITLE
Problem install versionPress github

### ISSUE THREAD

**Issue by Trecy-Acta** [versionpress/support#167]
Please be specific when describing the issue. Also, these details will help:

- VersionPress version:  Version 4.0-alpha1
- WordPress version: dernière version
- OS, webserver, PHP: php 7.2.18
- Github version: 1.6.6

I installed versionpress on my client's site, but I can not link to my github repository. How do I do ?

**Comment by pavelevap** [comment]
Hi @Trecy-Acta,

I am not sure what does it mean "Link to my Github repository"? Are you using Github Desktop (1.6.6) and do you want to open website tracked by VersionPress with this software? Your website is hosted on some hosting server and do you want to open it locally on your computer?

Do not forget to try the latest `4.0-beta2` version ([download](https://github.com/versionpress/versionpress/releases)).

**Comment by Trecy-Acta** [comment]
Yes, i use desktop github and I want to open website tracked by versionpress

[http://www.acta-informatique.fr/images/LogoAI2015baseline.png]<http://www.acta.asso.fr/>

Trécy Fournier
Développeuse
Tél. +33 1 40 04 52 66
Mobile : +33 6 01 31 19 92
trecy.fournier@acta-informatique.fr<mailto:prenom.nom@acta-informatique.fr/>

149 rue de Bercy, F-75595 PARIS cedex 12
Tél. +33 1 40 04 50 35
www.acta-informatique.fr<http://www.acta-informatique.fr/>

De : pavelevap <notifications@github.com>
Envoyé : mardi 28 mai 2019 20:33
À : versionpress/support <support@noreply.github.com>
Cc : Trècy FOURNIER <trecy.fournier@acta-informatique.fr>; Mention <mention@noreply.github.com>
Objet : Re: [versionpress/support] Problem install versionPress github (#167)


Hi @Trecy-Acta<https://github.com/Trecy-Acta>,

I am not sure what does it mean "Link to my Github repository"? Are you using Github Desktop (1.6.6) and do you want to open website tracked by VersionPress with this software? Your website is hosted on some hosting server and do you want to open it locally on your computer?

Do not forget to try the latest 4.0-beta2 version (download<https://github.com/versionpress/versionpress/releases>).

**Comment by pavelevap** [comment]
You can copy/clone Git repository created by Versionpress to your local computer and open it with Github Desktop (I am not sure if direct FTP/SFTP link will work directly in this case).

Please see our [article](https://versionpress.com/blog/2016/11/remote-workflows/) related to remote workflows, especially first example (creating Github/Bitbucket private repo, pushing site to it and then you can clone it and work with it).
